src/test/java/com/sap/sgs/phosphor/fosstars/nvd/NVDTest.java

```diff
@@ -103,6 +103,13 @@ public void matcher() throws IOException {
       entries = nvd.search(NvdEntryMatcher.entriesFor(new GitHubProject("openssl", "openssl")));
       assertNotNull(entries);
       assertEquals(3, entries.size());
+      
+      // Check if a reference URL can be found.
+      // TODO: The assert value has to be changed to 1. Once the issue
+      // https://github.com/SAP/fosstars-rating-core/issues/243 is fixed.
+      entries = nvd.search(NvdEntryMatcher.entriesFor(new GitHubProject("bcgit", "bc-java")));
+      assertNotNull(entries);
+      assertEquals(0, entries.size());
```

Comment:
According to the comment, this test case is wrong. Let's please remove it then.

Reply:
I will remove this then. I will have to revert the changes in `NVD_part.json`